container/lxd/initialisation_test.go

```diff
@@ -327,7 +333,7 @@ func (s *InitialiserSuite) TestFindAvailableSubnetUpperBoundAndLowerBoundInUse(c
 	})
 	subnet, err := findNextAvailableIPv4Subnet()
 	c.Assert(err, gc.IsNil)
-	c.Assert(subnet, gc.Equals, "1")
+	c.Assert(subnet, gc.Equals, "4")
 }
```

Comment:
Can you include a test like the one I described/the one in the bug? Something greater than a /24 like a /20.

One concern is that by only randomizing the 3rd octet anyone working with 10.0.0.0/16 will block all possible addresses.